src/servers/Cargo.toml

```diff
@@ -153,6 +154,7 @@ table.workspace = true
 tempfile = "3.0.0"
 tokio-postgres = "0.7"
 tokio-postgres-rustls = "0.12"
+moka = { workspace = true, features = ["sync"] }
```

Comment:
I think we don't need this dep at all. Just use the original HashMap, and check the capacity before inserting.